src/simulation/thermal/sensorThermal/sensorThermal.rst

```diff
@@ -0,0 +1,123 @@
+provides information on what this message is used for.
+
+.. list-table:: Module I/O Messages
+    :widths: 25 25 50
+    :header-rows: 1
+
+    * - Msg Variable Name
+      - Msg Type
+      - Description
+    * - sunInMsg
+      - :ref:`SpicePlanetStateMsgPayload`
+      - Input planet state message for the sun.
+    * - stateInMsg
+      - :ref:`SCStatesMsgPayload`
+      - Input spacecraft state input message.
+    * - sunEclipseInMsg
+      - :ref:`EclipseMsgPayload`
+      - Optional eclipse input message
+    * - sensorStatusInMsg
+      - :ref:`DeviceStatusMsgPayload`
+      - Optional device status input message
+    * - temperatureOutMsg
+      - :ref:`TemperatureMsgPayload`
+      - Output temperature message.
+
+Detailed Module Description
+---------------------------
+
+This module simulates the temperature of a sensor impacted by radiative thermal emission, radiative thermal absorption, and
+electrical power converted to thermal power.
+
+The radiative thermal emission is computed using the Stefan-Boltzmann Law:
+
+.. math::
+    \dot{Q}_{\text{emit}} = \epsilon \cdot \sigma \cdot A \cdot T^4
+
+where :math:`\epsilon` is the emissivity constant, :math:`\sigma` is the Stefan-Boltzmann constant, :math:`A` is the
+radiative surface area, and :math:`T` is the current temperature of the sensor.
+
+The radiative thermal absorption, which comes only from the sun in this module, is given by:
+
+.. math::
+    \dot{Q}_{\text{absorb}} = shadowFactor \cdot \alpha \cdot S \cdot A_{\text{proj}}
+
+The :math:`shadowFactor` is an indication of whether or not the sensor is in eclipse. A shadowFactor of 0 indicates the sensor
+is in eclipse. Likewise, a shadowFactor of 1 indicates the sensor is not in eclipse. The :math:`\alpha` parameter
+is the absorptivity constant of the sensor. :math:`S` is the solar constant, and :math:`A_{\text{proj}}` is the projected
+area of the sensor with respect to incoming solar flux.
+
+To compute the temperature at each timestep, the following energy balance equation is used:
+
+.. math::
+    \rho \cdot Vol \cdot c_p \dfrac{dT}{dt} = \dot{Q}_{\text{absorb}} + \dot{Q}_{in} - \dot{Q}_{\text{emit}}
+
+where :math:`\rho` is the density of the material, :math:`Vol` is the volume of the material, and :math:`c_p` is the
+specific heat. :math:`\dot{Q}_{in}` is the electrical power consumed by the sensor.
+
+The change in temperature is solved for, and the current temperature is updated using Euler integration:
+
+.. math::
+    T_i = T_{i-1} + \dfrac{dT}{dt} \cdot dt
+
+
+Model Assumptions and Limitations
+---------------------------------
+
+This code makes the following assumptions:
+
+- The sensor is a flat plate with an insulated backing (i.e. there is no conduction from the sensor to the spacecraft).
+- The Earth's albedo does not impact the temperature.
+- All of the electrical power is converted to heat. The heat is conducted throughout the sensor instantaneously.
+
+
+User Guide
+----------
+
```

Comment:
You have the module use default values for some parameters.  Please list all parameters that have default values, and explain what parameters must be set.

Reply:
Made the suggested change with a table in the sensorThermal documentation.